frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java

```diff
@@ -49,23 +43,7 @@ public GitMachetePanel(Project project) {
     val selectionComponent = selectedGitRepositoryProvider.getSelectionComponent();
     val graphTable = getGraphTable();
 
-    val messageBusConnection = project.getMessageBus().connect();
```

Comment:
😯 huh if you could extract changes unrelated to the feature in question to a separate PR 🙏🏻

Reply:
`git reset develop && git add -p` FTW

Reply:
(unless it's all non-obviously related in fact 🕵🏻)